src/mmrelay/db_runtime.py

```
@@ -0,0 +1,186 @@
+        conn = sqlite3.connect(self._path, check_same_thread=False)
+        try:
+            if self._busy_timeout_ms:
+                conn.execute(f"PRAGMA busy_timeout={int(self._busy_timeout_ms)}")
+            if self._enable_wal:
+                # journal_mode pragma returns the applied mode; ignore result
+                conn.execute("PRAGMA journal_mode=WAL")
+            conn.execute("PRAGMA foreign_keys=ON")
+            for pragma, value in self._extra_pragmas.items():
+                # Validate pragma name to prevent injection.
+                if not re.fullmatch(r"[a-zA-Z_]+", pragma):
+                    raise ValueError(f"Invalid pragma name provided: {pragma}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)